teuthology/task/install/rpm.py

```diff
@@ -126,6 +126,7 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
     rpm = _package_overrides(rpm, remote_os)
 
     dist_release = remote_os.name
+    dist_major_version = int(str(remote_os.version).split(".")[0])
```

Comment:
I wonder should we treat remote_os.version to be a string? If so, then we should not use 'str' conversion, and fix the bug when it different type like int or float.

Reply:
yeah, if this line doesn't work that would indicate terrible breakage indeed